test/functional/api/v2/discovered_hosts_controller_test.rb

```diff
@@ -29,6 +29,9 @@ def switch_controller(klass)
                        :name => 'discovery_reboot',
                        :value => true,
                        :category => 'Setting::Discovered')
+
+    #load the power.rb file, as PowerService is not autoloaded from file named power
+    ForemanDiscovery::NodeAPI::Power 
```

Comment:
we can remove this line once https://github.com/theforeman/foreman_discovery/pull/229 is merged